src/Components/inputs/Story.jsx

```diff
@@ -1,40 +1,61 @@
 import React from 'react';
 
-const TextInput = () => {
+const TextInput = ({getTitle}) => {
   return (
     <> 
       <form>
-        <input type='text' />
+        <input
+          type='text' 
+          onChange={(e) => {
+            getTitle(e.target.value)
+          }}
```

Comment:
components should be lean as much as possible. this looks messy. do the slicing inside `getTitle` so all you have here is `onChange={getTitle}`.